taskcluster/fenix_taskgraph/transforms/multi_dep.py

```diff
@@ -24,7 +26,7 @@ def build_name_and_attributes(config, tasks):
         }
         primary_dep = task["primary-dependency"]
         copy_of_attributes = primary_dep.attributes.copy()
-        task.setdefault("attributes", copy_of_attributes)
+        task.setdefault("attributes", {}).update(copy_of_attributes)
```

Comment:
It seems surprising that we'd overwrite any attributes explicitly set on the task, no?